code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -105,9 +105,7 @@
 
 /mob/living/carbon/human/proc/check_block()
 	if(mind)
-		if(mind.martial_art && mind.martial_art.block_chance \
-		&& prob(mind.martial_art.block_chance) && in_throw_mode \
-		&& !stat && !knockdown && !stun)
+		if(mind.martial_art && mind.martial_art.block_chance && prob(mind.martial_art.block_chance) && in_throw_mode && !stat && !knockdown && !IsStun())

```

Comment:
holy crap, that's a long boolean operation.  Can be slightly shortened though, as the block_chance is made redundant by the prob

`if(mind.martial_art &&  prob(mind.martial_art.block_chance) && in_throw_mode && !stat && !knockdown && !IsStun())`